app.js

```diff
@@ -121,14 +134,45 @@ router.post('/contract/view/:name/:methodName', async ctx => {
 
 router.get('/account/:name', async ctx => {
     const response = await client.request('view_account', [{
-        account_id: await hash(ctx.params.name),
+        account_id: await encodeAccountNameForRpc(ctx.params.name),
         method_name: '',
         args: []
     }]);
     checkError(ctx, response);
     ctx.body = response.result;
 });
 
+/**
+ * Create a new account. Generate a throw away account id (UUID).
+ */
+router.post('/account', async ctx => {
+    // TODO: this is using alice account to create all accounts. We may want to change that.
+    const nonce = await getNonce(ctx, defaultSender);
+    ensureTrue(nonce);
```

Comment:
1. You can use `ctx.assert(nonce)` if needed, no need for custom helper.
2. Why is it needed? Are there any cases when `view_account` doesn't return `nonce` without error?

Reply:
This is more for code readability - it spells out that we are specifically looking for non empty nonce.